WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewModel.swift

```diff
@@ -601,6 +601,12 @@ extension ProductFormViewModel {
                 }
             }
         case .edit:
+            guard hasChangesExcludingImageUploads() else {
```

Comment:
I wonder if this case should be unit tested, seems like something that is easy to forget in the future and can easily introduce regression.

Reply:
Yup that would be ideal but I tried and it required more changes than expected as `ProductFormRemoteActionUseCase` was not injected so not testable. I can create a GH issue for improving if necessary.